sample/android/app/build.gradle

```diff
@@ -165,13 +165,6 @@ android {
         }
     }
 
-    packagingOptions {
-        pickFirst "lib/armeabi-v7a/libc++_shared.so"
-        pickFirst "lib/arm64-v8a/libc++_shared.so"
-        pickFirst "lib/x86/libc++_shared.so"
-        pickFirst "lib/x86_64/libc++_shared.so"
-    }
```

Comment:
M: this is not needed anymore or what?

Reply:
this is not needed, you can check in RN Upgrade Tool the differences.

Reply:
It seems to have been removed here:
<img width="1362" alt="Screen Shot 2020-07-21 at 6 23 34 PM" src="https://user-images.githubusercontent.com/30991498/88049408-4c52c780-cb7f-11ea-919f-c739ecd67967.png">

 I can't find a way to link to the exact line/file: https://react-native-community.github.io/upgrade-helper/?from=0.62.2&to=0.63.1

Reply:
thx